### ISSUE TITLE
1.19 update in this mod with new structure then

### ISSUE THREAD

**Issue by ENUSSOUL001** [TelepathicGrunt/RepurposedStructures#211]
seems like 1.19 is out now so what idea structure do you added(plan)? well for me i want ancient city village in deep dark biome just a susgestion

Label: Susgestion

**Comment by TelepathicGrunt** [comment]
1.19 just came out lol. It's going to take me a while to update my mods to 1.19 first. get them stable. And then with any remaining free time, maybe i'll look into new structures. But a deep dark village is not going to happen as shulk kills villagers and it doesn't make sense for villagers to be so far down.

**Comment by ENUSSOUL001** [comment]
i mean a ancient city village with warden is a residence

**Comment by TelepathicGrunt** [comment]
closing this as I added some structures to https://github.com/TelepathicGrunt/RepurposedStructures-Fabric/issues/4 that I would look into to see if they will look nice or not. But that's a long term goal as porting and compat datapacks are my main priority.

 I also decided against adding any structure to the deep dark now that i got a good look at it. I couldn't tell where the regular deepslate areas start and where the deep dark start. It just looks like the regular underground with patches of shulk. therefore, it doesn't seem distinct enough for me to anything too it and not just look like a regular stone structure with shulk patches all over. The existing underground structures I have currently (underground bastion) already looks fine as is in deep dark.
